source/_statement.html.md

```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
 The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the [Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
 
 ## Technical information about this service’s accessibility
-We're committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+The CDIO is committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
 
 This service is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, because of the non-compliances listed below.
 
 ## Non accessible content
 
-Some content on our [status page](https://status.wifi.service.gov.uk/) is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances are: 
-
-* an incorrect heading hierarchy, which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)
-
-* missing level 1 headings, so users may not be able to accurately determine the structure of content on the page
+### PDFs and other documents
 
-* a missing main heading and an incorrect heading hierarchy on the status home page, which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships)
+One page links to a PDF document that is not fully accessible. This is not a fail under any WCAG 2.1 success criterion.
```

Comment:
As suggested earlier, we don't 'link' to a PDF, we own and upload the PDF to the service. 

So I'd recommend "Our 'Memorandum of Understanding' is a PDF document..." etc